pkg/arvo/sys/hoon.hoon

```diff
@@ -5605,6 +5605,15 @@
 ++  but  (cook |=(a/@ (sub a '0')) (shim '0' '1'))      ::  binary digit
 ++  cit  (cook |=(a/@ (sub a '0')) (shim '0' '7'))      ::  octal digit
 ++  dem  (bass 10 (most gon dit))                       ::  decimal to atom
+++  dsm                                                 ::  decimal to signed int
+  %+  cook
+    |=  [s=? n=@]
+    ^-  @s
+    ?:  s
+      (mul n 2)
+    (dec (mul n 2))
```

Comment:
You can replace this with just `new:si`, which is the existing stdlib function for "unsigned to signed", and matches the function signature used here.